src/components/TestDetailsDialog/TestDetailsModal.tsx

```diff
@@ -269,11 +282,11 @@ const TestDetailsModal: React.FunctionComponent<{
   const saveIgnoreAreasOrCompareArea = () => {
     if (valueOfIgnoreOrCompare.includes("Ignore")) {
       saveTestRun(ignoreAreas, "Ignore areas are updated.");
-    } else {
+    } else if (currentImage) {
       const invertedIgnoreAreas = invertIgnoreArea(
-        image!.width,
-        image!.height,
-        head(ignoreAreas),
+        currentImage.width,
+        currentImage.height,
+        ignoreAreas[0],
```

Comment:
this will throw eslint error 
use `head` to avoid it

Reply:
I removed the use of `lodash` as it was not as a dependency. I will see if linter gives me issues

Reply:
Cannot get linter error here, what are you seeing?